lib/controllers/widget/device_list.dart

```diff
@@ -110,13 +108,14 @@ class _DeviceListState extends State<DeviceList> {
     setState(() {
       _loading = true;
     });
-    await Devices.getList(_authToken, _list.nextPage).then((value) {
+    await Devices.getList(_authToken, _list.nextPageToken).then((value) {
       setState(() {
         value.chromeosdevices.forEach((element) {
-          _list.chromeosdevices.add(element);
+          _list = _list.rebuild((b) => b.chromeosdevices..add(element));
```

Comment:
2 dots?

Reply:
It is a cascade notation, works same as dot but let you do more operations.